### PR TITLE
PAN-OS - tags UC

### PR DESCRIPTION
**Pull request by bakatzir**
## Status
Ready

## Related Issues
fixes: https://github.com/demisto/etc/issues/18816
fixes: https://github.com/demisto/etc/issues/18783 - @kirbles19 

## Description
added full support in the tags mechanism, please see referenced issue for full details.

## Does it break backward compatibility?
   - No

## Must have
- [x] Tests
- [x] Documentation - after product-docs team review
- [x] Code Review

### CODE REVIEW

**Comment by bakatzir** [comment]
@kirbles19 
changelog:
https://github.com/demisto/content/blob/479a1cc1f571d431d373fcbebc2b497525810093/Integrations/Panorama/CHANGELOG.md
yml:
https://github.com/demisto/content/blob/479a1cc1f571d431d373fcbebc2b497525810093/Integrations/Panorama/Panorama.yml

**Comment by richardbluestone** [comment]
I've updated the yml file. Note that in the panorama-get-pcap output context output, File extension is wrongly spelled as File.Extenstion.

**Comment by yuvalbenshalom** [comment]
@anara123 or @glicht Please force merge this.
It is only failed on DAG config test that is flaky anyway on master and we are going to skip it today anyway.

File: Integrations/Panorama/CHANGELOG.md [hunk 1]
```diff
@@ -1,8 +1,12 @@
 ## [Unreleased]
+- Added the *tag* argument to several commands.
+    - List commands - filter by a tag.
+    - Create and edit commands
+    - Added the context output Tags to all list, create, edit, and get commands.
   - Added support in the ***panorama-query-logs*** command to supply a list of arguments, which are separated using the "OR" operator.
-  - Improved error messaging when trying to configure a device-group that does not exist.
-
-## [19.9.0] - 2019-09-04
```
Comment (glicht) [inline Integrations/Panorama/CHANGELOG.md]: Bring back the date.

**Pull request merged**